package.json

```diff
@@ -32,7 +32,9 @@
   "dependencies": {
     "archiver": "2.0.0",
     "bluebird": "3.4.6",
+    "jszip": "^3.1.4",
```

Comment:
Please pin this at a specific version, npm is terrible.

Reply:
Yes it is, I thought I set `npm` to avoid the `^` but apparently I haven't